### PR TITLE
fix(navbar): centred env icon

### PR DESCRIPTION
**Pull request by denkristoffer**
# Purpose of PR

Centred environment icon

### CODE REVIEW

**Review by Lelith — APPROVED** [review]

File: packages/components/navbar/src/NavbarSwitcher/NavbarEnvVariant.tsx [hunk 1]
```diff
@@ -12,19 +12,11 @@ export function NavbarEnvVariant({
...
-      }
-    >
-      {isAlias ? (
-        <EnvironmentAliasIcon size="tiny" />
-      ) : (
-        <EnvironmentIcon size="tiny" />
-      )}
-    </span>
+  return isAlias ? (
+    <EnvironmentAliasIcon color={color} size="tiny" />
```
Comment (Lelith) [inline packages/components/navbar/src/NavbarSwitcher/NavbarEnvVariant.tsx:18]: are these not still old icons?

Comment (denkristoffer) [reply packages/components/navbar/src/NavbarSwitcher/NavbarEnvVariant.tsx:18]: No they are from the alpha: https://github.com/contentful/forma-36/blob/afc9f97d979e6e516c63b3f43d44348d70b629db/packages/components/navbar/package.json#L14

File: packages/components/navbar/src/NavbarSwitcher/NavbarSwitcher.tsx
Comment (cf-remylenoir) [inline packages/components/navbar/src/NavbarSwitcher/NavbarSwitcher.tsx:1]: I can't seem to be able to use `ENTER` to activate the button

Comment (Lelith) [reply packages/components/navbar/src/NavbarSwitcher/NavbarSwitcher.tsx:1]: i think in storybook there is no on-click action behind the switcher

**Review by cf-remylenoir — APPROVED** [review]

**Pull request merged**